<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>31</y>
     <width>195</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="DieThrowlabel">
        <property name="text">
         <string>Die throw</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DieThrowcomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Ruleslabel">
        <property name="text">
         <string>Rules                  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="RulescomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="Themeslabel">
        <property name="text">
         <string>Themes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ThemescomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Soundslabel">
        <property name="text">
         <string>Sounds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="SoundscomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="StartButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BackButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BackButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
